src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ please check the latest changes
 
 ## Unreleased
 
+* `TracerProviderBuilder`'s [AddLegacySource()](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Trace/TracerProviderBuilderBase.cs#L171)
```

Comment:
Adds `AddLegacySource()` method to `TracerProviderBuilder` - just this should be good. I'd suggest to avoid linking to source code in changelogs.

Reply:
ALso a similar entry needed in changelog for OpenTelemetry SDK project as well.